scriptmodules/ports/abuse.sh

```diff
@@ -10,8 +10,9 @@
 #
 
 rp_module_id="abuse"
-rp_module_desc="Abuse"
+rp_module_desc="Abuse - port of run and gun game"
 rp_module_license="GPL https://raw.githubusercontent.com/Xenoveritas/abuse/master/COPYING"
```

Comment:
This isn't your fault, but if you do get around to updating the PR with Jools' other comments, it would be helpful if you fixed this typo as well. The function name is supposed to be the non-US spelling (`rp_module_licence`). Thanks.